site/content/entry/yuzu-patreon-preview-sept-2019/index.md

```diff
@@ -0,0 +1,93 @@
++++
+date = "2019-09-09T00:55:00+05:30"
+title = "yuzu Patreon Preview Release September 2019"
+author = "CaptV0rt3x"
+forum = 144316
++++
+
+Good Day yuz-ers! (*yeah, that's sticking!*)
+Many of you thought that yuzu's development slowed down, didn't you?
+Well, guess again - The team has been working tirelessly to bring you this exciting new feature!
+What is it? Jump right in to find out!
+<!--more-->
+****
+{{< single-title-imgs
+    "Super Mario Maker 2 - Rescaled to 6k"
+    "./smm2-1.jpg"
+    "./smm2-2.jpg"
+    "./smm2-3.jpg"
+>}}
+
+We hope everyone had an enjoyable and relaxing summer.
+While you may have noticed a bit of a lull during August, make no mistake that our team was still hard at work!
+Behind the scenes, our GPU team has been investing a ton of time and talent into bringing about this patreon releases' biggest feature: `Resolution Rescaler!`
+
+### You can download this release on our [Patreon](https://www.patreon.com/yuzuteam)!
+
+As you might expect from the name, this feature increases native resolution through a set of rules enforced by a profile, very similar to other emulators such as Cemu.
+You'll notice an option to `Enable Resolution Scanner` in your graphics settings menu.
```

Comment:
This is not an option anymore, instead in the resolution drop down box you can decide wether you're upscaling or not. If you're not then you're building the rescale profile. So the option to enable or disable the resolution scanner isn't there anymore.